homeassistant/components/matter/__init__.py

```diff
@@ -226,30 +225,46 @@ async def async_remove_entry(hass: HomeAssistant, entry: ConfigEntry) -> None:
         LOGGER.error(err)
 
 
+def _remove_via_devices(
+    hass: HomeAssistant, config_entry: ConfigEntry, device_entry: dr.DeviceEntry
+) -> None:
+    """Remove all via devices associated with a device."""
+    device_registry = dr.async_get(hass)
+    devices = dr.async_entries_for_config_entry(device_registry, config_entry.entry_id)
+    for device in devices:
+        if device.via_device_id == device_entry.id:
+            LOGGER.info("Removing device %s", device.id)
```

Comment:
Why is this logged at info level? The device id isn't user facing information.

Reply:
Hm, yeah that one wasn't intentional and slipped through.

Addressed with: https://github.com/home-assistant/core/pull/122036